Lib/test/certdata/make_ssl_certs.py

```diff
@@ -266,6 +283,11 @@ def write_cert_reference(path):
         f.write(key)
         f.write(cert)
 
+    cert = extract_cert(cert, 0)
+    with open('cert3.pem', 'w') as f:
+        f.write(key)
```

Comment:
This should be removed. Then the test passes.

Reply:
Ouch, that was a stupid copy'n'paste mistake...
```suggestion
```